coriolis/osmorphing/osmount/windows.py

```diff
@@ -105,6 +110,30 @@ def _service_disks_with_status(
                             raise
                     break
 
+    def _get_disk_ids_from_drive_letters(self, drive_letters):
+        disk_ids = []
+        volume_entry_re = r"\s+Volume ([0-9]+)\s+([A-Z]+)\s+"
+        volume_list_script = "LIST VOLUME\r\nEXIT"
+        vol_list = self._run_diskpart_script(volume_list_script)
+
+        def get_disk_ids(vol_id):
```

Comment:
Stylistic yet ultimately pointless nit: please prefix the function name with `_` since it technically counts as "private to this scope"